documentation/site/content/samples/azure-kubernetes-service/includes/create-aks-cluster-storage.txt

```diff
@@ -184,91 +181,65 @@ You will dynamically create and use a persistent volume with Azure Files NFS sha
-$ ./create-domain-on-aks.sh -i my-create-domain-on-aks-inputs.yaml -o ~/azure -u ${TIMESTAMP}
-```
-
-After running the command, all needed configuration files are generated and output to `~/azure/weblogic-on-aks`:
+##### Generated configuration files
+Use below command to generate configuration files.
 
 ```shell
-The following files were generated:
-  /home/username/azure/weblogic-on-aks/azure-csi-nfs.yaml
-  /home/username/azure/weblogic-on-aks/pvc.yaml
-  /home/username/azure/weblogic-on-aks/admin-lb.yaml
-  /home/username/azure/weblogic-on-aks/cluster-lb.yaml
-  /home/username/azure/weblogic-on-aks/domain1.yaml
-
-Completed
-```
-
-**NOTE**: Beyond the required and default configurations generated by the command, you can modify the generated YAML files to further customize your deployment. For further information about customizing your deployment, consult the operator documentation, [AKS documentation](https://docs.microsoft.com/azure/aks/), and Kubernetes references.
-
-##### Apply generated configuration files
-
-To define how an Azure file NFS share is created, we have provided a configuration file `azure-csi-nfs.yaml`. You can find it in your output directory.
-
-The following command displays the YAML content of your current settings. This should be the same with content in `azure-csi-nfs.yaml`. The Storage Class name is `azurefile-csi-nfs`.
+cat >azure-csi-nfs-${TIMESTAMP}.yaml <<EOF
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-This sample uses Azure Files Container Storage Interface (CSI) drivers to manage the NFS file share, provisioner is `file.csi.azure.com`, see the [Azure Files CSI drivers documentation](https://docs.microsoft.com/azure/aks/azure-files-csi).
-
-```shell
-cat <<EOF
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: azurefile-csi-nfs
 provisioner: file.csi.azure.com
-allowVolumeExpansion: true
 parameters:
   protocol: nfs
-mountOptions:
-  - nconnect=4
+  resourceGroup: ${AKS_PERS_RESOURCE_GROUP}
+  storageAccount: ${AKS_PERS_STORAGE_ACCOUNT_NAME}
+  shareName: ${AKS_PERS_SHARE_NAME}
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+allowVolumeExpansion: true
+
+EOF
+
+cat >pvc-${TIMESTAMP}.yaml <<EOF
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wls-azurefile-${TIMESTAMP}
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile-csi-nfs
+  resources:
+    requests:
+      storage: 5Gi
+
 EOF
 ```
 
 Use the `kubectl` command to create the Storage Class and persistent volume claim to the `default` namespace.
 
 ```shell
-$ kubectl apply -f ~/azure/weblogic-on-aks/azure-csi-nfs.yaml
-```
-
-```
-persistentvolume/wls-azurefile created
+$ kubectl apply -f azure-csi-nfs-${TIMESTAMP}.yaml
+$ kubectl apply -f pvc-${TIMESTAMP}.yaml
 ```
 
 Use the following command to verify:
 
 ```shell
-$ kubectl get sc
+$ kubectl get pvc
```

Comment:
The above commands in 229-230 create both sc and pvc,   shouldn't we check both withe kubectl get ? (sc is removed)

Reply:
Yes, it's better to check both, will update.